### PR TITLE
Update dependencies and generated files

### PR DESCRIPTION
**Pull request by rmsyn**
Updates cargo dependencies to their latest versions, and runs the `update.sh` script to generate updated source files from the SVD file.

### CODE REVIEW

**Comment by Disasm** [comment]
Please update the versions of software you used in `update.sh`

File: Cargo.toml [hunk 1]
```diff
@@ -10,9 +10,15 @@ license = "ISC"
 edition = "2018"
 
 [dependencies]
-bare-metal = "0.2.4"
-riscv = "0.6.0"
-vcell = "0.1.2"
+bare-metal = "1.0.0"
```
Comment (Disasm) [inline Cargo.toml]: Apparently `bare-metal` is not used anymore.

Comment (rmsyn) [reply Cargo.toml]: Ok, I can remove it.

**Review by Disasm — APPROVED** [review]
Thank you!

**Comment by Disasm** [comment]
Anything else you would like to add before a release?

**Comment by rmsyn** [comment]
> Anything else you would like to add before a release?

Not that I can think of, at the moment.

**Pull request merged**